client/landing/gutenboarding/components/mshots-image/index.tsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+import classnames from 'classnames';
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface MShotsImageProps {
+	url: string;
+	alt: string;
+	'aria-labelledby': string;
+	retryDelay: number;
+}
+
+export function mshotsUrl( url: string, count = 0 ): string {
+	const mshotsUrl = 'https://s0.wp.com/mshots/v1/';
+	const mShotsParams = {
+		// viewport size (how much of the source page to capture)
+		vpw: 1200,
+		vph: 3072,
+		// size of the resulting image
+		w: 700,
+		h: 1800,
```

Comment:
One thing to try is put ridiculously low jpg quality, and pretty big physical image size.

When rendered in small enough canvas, it sometimes looks better and loads faster than better quality, but smaller physical size image. 😄 

Unless mshots is smart and returns webp for browsers that support it? 🤔